README.md

```diff
@@ -2,8 +2,12 @@
 
 [![Join the chat at https://gitter.im/mozillascience/software-citation-tools](https://badges.gitter.im/mozillascience/software-citation-tools.svg)](https://gitter.im/mozillascience/software-citation-tools?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-The Software Citation Tools will be a suite of tools for the scientific community to assist them in citing software that is found online and used in research and scientific papers.  The purpose of this is to reduce friction for researchers to cite software and make sure that software developers are given the credit they are due.  The suite will be developed in conjunction with Mozilla Science Lab via Abigail Cabunoc Mayes, the scientific community, and the open source software development community.  The project will require extensive interviews with members of the research community to gather information on how they are currently citing software, and how these tools can streamline that process.
+The Software Citation Tools is a package of tools written in javascript and made available through the npm javascript package manager. The main part of the package will be a library which will create a citation when given a link to a github repository. Further tools will be created on top of this library to make using this library easier and extend it's usefulness.
```

Comment:
Just a few grammatical things:
- if we're going to use Software Citation Tools as a title, as we discussed on Thursday, it should be "Software Citation Tools is" not "The Software Citation Tools is"
- "javascript" should be "JavaScript"
- "github" should be "GitHub"
- "it's usefulness" should be "its usefulness"